### ISSUE TITLE
intervalOptions does not exist after Craft v3 → v4 upgrade

### ISSUE THREAD

**Issue by iparr** [putyourlightson/craft-campaign#336]
I'm moving from Craft CMS v3.7 to 4.2.8 with Campaign Pro v2.1.11

Everything is going okay, but when I view one of my weekly 'Recurring' emails (`https://whatever.test/admin/campaign/sendouts/recurring/preview/427652`) I get:

```
Twig Runtime Error – [Twig\Error\RuntimeError](http://twig.sensiolabs.org/api/2.x/Twig/Error/RuntimeError.html)
Variable "intervalOptions" does not exist.
```

I've tried removing all content from my email templates, but this has no impact. The error is with Campaign templates:

"1. in /Users/ian/Sites/whatever/vendor/putyourlightson/craft/campaign/src/templates/sendouts/_includes/schedule/recurring.twig -  at line 22":  
`{{ schedule.frequency }} {{ intervalOptions[schedule.frequencyInterval] }}`

This error was not here with the previous version of Craft CMS and Campaign. I don't know what this field is.

**Comment by bencroker** [comment]
Thanks for reporting this! Fixed in https://github.com/putyourlightson/craft-campaign/commit/351387482cc9a61f373d47910cabe78ea6c44e18 and released in version 2.1.12.

**Comment by iparr** [comment]
Thank you. I can confirm that this has fixed my scenario.
